app/settings.py

```diff
@@ -40,15 +40,16 @@
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
-    # 'django.contrib.staticfiles',
+    "django.contrib.staticfiles",
+    "django.contrib.admin",
     "rest_framework",
     "rest_framework_json_api",
     "addon_service",
 ]
 
 if DEBUG:
     # run under ASGI locally:
-    INSTALLED_APPS.append("daphne")  # django's reference asgi server
+    INSTALLED_APPS.insert(0, "daphne")  # django's reference asgi server
```

Comment:
I know @mfraezz decided to go with daphne for the actual deployment. Maybe check and see if this can be moved out of the DEBUG?

Reply:
i thiiink the only reason to mention `daphne` in app settings is to use it for `manage.py runserver` (an explicitly not-for-production dev tool), and we should avoid assuming a specific webserver (even when we know)